frontend/src/components/CalendarView/CalendarView.jsx

```diff
@@ -99,6 +99,10 @@ function CalendarView () {
       })
       setIsModalOpen(true);
     }
+
+    const handleEventSelect = (info) => {
```

Comment:
I assume you will implement this later on. It is a good practice to add a `TODO: this is going to implement in the next commit` as a comment, so the reviewer knows that you will implement this in the next commit